frontend/src/app/static/page-404/page-404.component.scss

```diff
@@ -0,0 +1,5 @@
+.wrapper {
+    min-height: 70vh;
```

Comment:
what's the rationale for such height?

Reply:
Our "main-content" is set to `70vh` in `app.component.scss`, I wanted to put "przepraszamy..." in the center to make it look better for now. Design of this component will be discussed on Wednesday.